input/pages/client-confidential-asymmetric.md

```diff
@@ -280,7 +280,7 @@ At runtime, when the bilirubin monitoring service needs to authenticate to the t
 
 ```
 {
-  "iss": "https://bili-monitor.example.com",
+  "iss": "bili-monitor",
```

Comment:
```suggestion
  "iss": "https://bili-monitor.example.com",
```

What was the intention behind changing the `iss`? The previous value is consistent with the value used to caluculate signatures in the JWT examples and reflects a full URL, as expected in real applications

Reply:
it is documented that that are the same see my comments above -  this is confusing .

![image](https://user-images.githubusercontent.com/7410922/139951195-1b5f0edd-f1c3-4169-8ef6-6e69e53b9d7c.png)

Reply:
Gotcha. The example needs to be re-worked here. Will start with https://bili-monitor.example.com for both.